src/osx.md

```diff
@@ -1,12 +1,17 @@
 # Mac OS X
 
-For now, the easiest way to get going is to install Linux on a virtual
-machine, and follow the [Linux instructions](./linux.html).
+The tools you need are similar to the tools listed in the [Linux instructions](./linux.html), but we will need to build a cross compiler in order to get things working. This is sort of complicated and boring, but we've done some of that boring work for you so you can get up and running quickly.
 
-Otherwise, to install the other tools on Mac OS X, make sure to have [homebrew](http://brew.sh/) installed.
+Make sure you have [homebrew](http://brew.sh/) installed because you are going to need it to install some of the tools. You are probably also going to need [Xcode](https://developer.apple.com/xcode/download/) if you don't already have it.
```

Comment:
Is it possible to make a note that once Xcode is downloaded we make sure they open the program to accept the agreement/popup. Do we also need to tell people to install the Xcode command line tools? I can't remember if it happens automatically after accepting the agreement/popup on first open. Can we also explain why it's needed? Just because I remember myself bashing my head against my computer when I had errors in the command line and it was all because I didn't open Xcode! :grimacing: